week4/file-management/cifar10-file-management/scripts/train_with_config.py

```
@@ -1,74 +1,117 @@
+        epochs=config.epochs,
+        validation_data=(x_test, y_test),
+        verbose=1,
+    )
+
+    model_path = Path(model_path)
+    model_path.parent.mkdir(parents=True, exist_ok=True)
+    model.save(model_path)
+    print(f"\nModel saved to {model_path}")
+
+    print(f"\nFiles in {model_path.parent}:")
+    os.system(f"ls -la {model_path.parent}")
```

Ruff (0.16.1)

[error] Starting a process with a shell, possible injection detected

(S605)